index.html

```diff
@@ -0,0 +1,1704 @@
+                            <div class="category">
+                                <span>DOWNTIME</span>
+                                <span>CULTURE</span>
+                            </div>
+                            <h2><a class="links" href="#">Lorem ipsum dolor sit amet consectetur adipisicing elit.
+                                    Eveniet, mollitia iure? Quidem adipisci architecto suscipit?</a></h2>
+                        </article>
+                    </div>
+                    <div class="col-12 col-md-3">
+                        <article class="magazine-stories">
+                            <a href="#"><img class="img-magazine"
+                                    src="https://d.newsweek.com/en/full/1551046/beer-alcohol-stock-getty.webp?w=397&h=265&f=6df0f9c87488c9a122f5d3ed5e02ec61"
+                                    alt="" /></a>
+                            <div class="category">
+                                <span>DOWNTIME</span>
+                                <span>CULTURE</span>
+                            </div>
+                            <h2><a class="links" href="#">Lorem ipsum dolor sit amet consectetur adipisicing elit.
+                                    Eveniet, mollitia iure?
+                                    Quidem adipisci architecto suscipit?</a></h2>
+                        </article>
+                    </div>
+                    <div class="col-12 col-md-3">
+
+                    </div>
+                </div>
+            </div>
+            <div class="grid-stories">
+                <div class="container-fluid">
+                    <div class="divider w-100 d-flex justify-content-center">
+                        <div class="newsweek-logo-s"></div>
+                        <span>EDITOR PICK</span>
+                    </div>
+                    <div class="row">
+                        <div class="col-12 col-md-6 col-lg-3 col-3-w">
+                            <article class="featured-stories-article">
+                                <div class="card">
+                                    <div class="image-in-card position-relative">
+                                        <a href="#"><img class="card-img-top"
+                                                src="https://d.newsweek.com/en/full/1551658/sneeze-cold-illness-flu-stock-getty.webp?w=397&h=265&f=e539b9b52e5961febb6e31862bdd673b"
+                                                alt="" /></a>
+                                        <span class="badge badge-light">WORLD</span>
+                                    </div>
+                                    <div class="card-body">
+                                        <h2 class="card-title">
+                                            <a class="links font-weight-bold" href="#">
+                                                Lorem ipsum dolor sit</a>
+                                        </h2>
+                       
```

Comment:
##### Footer

* [ ] It contains social media icons ([link to page](https://www.newsweek.com/))
* [ ] **It changes with RWD breakpoints** ([link to page](https://www.newsweek.com/))